src/react-components/src/__tests__/components/Label-test.tsx

```diff
@@ -24,7 +24,7 @@ describe("Label Test", () => {
   });
 
   it("if both <label> children and labelText prop are passed, use labelText", () => {
-    wrapper = Enzyme.shallow(<Label id="label" referenceId="so-lonely" labelText="cupcakes" >muffins</Label>);
+    wrapper = Enzyme.shallow(<Label id="label" referenceId="so-lonely" labelContent={<>cupcakes</>} >muffins</Label>);
```

Comment:
A story for this type would be good.

Reply:
Reminded me that I really don't like having to pass in content and also checking for children, and that it should be OK just to check for children.  This is a breaking change but this PR is full of them, so I made those updates